subsys/bluetooth/mesh/main.c

```diff
@@ -123,6 +123,10 @@ int bt_mesh_provision(const u8_t net_key[16], u16_t net_idx,
 
 	memcpy(bt_mesh.dev_key, dev_key, 16);
 
+	if (IS_ENABLED(CONFIG_BT_MESH_LOW_POWER)) {
+		bt_mesh_lpn_group_add(BT_MESH_ADDR_ALL_NODES);
+	}
+
```

Comment:
IMO, this functionality should be compile time configurable. The working group made it opt-in for a reason, it's very easy to flood the friend queue with broadcast packets that might not be relevant for the LPN

Reply:
Agreed, and I'd default to disabled for the new option (not just because it seems to be the safe choice but also to keep consistency with previous Zephyr versions).

Reply:
An alternative would be to expose the group add/del functions as public API (and clearly document that they should only be used for special cases like this, whereas the stack takes automatically care of the rest). Thoughts?

Reply:
Answering to myself: I'm mildly leaning toward the Kconfig since then the app can't mess around with what the stack manages automatically (e.g. remove groups that the stack had added, etc).